Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2410 -- Adjust display of public items in policy repository

Resolves [EREGCSC-2410](https://jiraent.cms.gov/browse/EREGCSC-2410)

**Description**

Display public items like internal items.

**This pull request changes:**

- Rolls back changes made in EREGCSC-2340 -- we want `summary_headline` to always be a normal summary headline field, and we want `content_headline` to be added back and always be a normal content headline.
- Rolls back the tests added for 2340
- Adds and refactors logic on the front end so that public items are displayed as internal items.

**Steps to manually verify this change:**

1. Visit [experimental deployment and search for "federal requirements"](https://qitnd6ajab.execute-api.us-east-1.amazonaws.com/dev1101/policy-repository/?q=%22federal%20requirements%22)
2. Make sure the associated public documents have a normal title link and a descriptive snippet under the title link
3. Clear the search text and make sure that public documents only have a normal title link
4. Make sure all other internal and public results look as expected
5. Make sure all tests pass (python, vitest, cypress)
